core/src/main/resources/org/springframework/ldap/config/spring-ldap-2.0.xsd

```diff
@@ -262,6 +262,209 @@
         </xs:attribute>
     </xs:attributeGroup>
 
+    <xs:attributeGroup name="pooling2.attlist">
```

Comment:
I tried the approach for transparent upgradation. Like if pool2 classes are there in CLASSPATH use pool2 otherwise fall back to pool1. But the main problem is that, they have somewhat different configuration. Pool1.x and Pool2.x has breaking changes. Not only package name changes, they have different class names, different pool configuration properties etc. So I could not find a way but to introduce a separate xml config. By this only I can ensure backward compatibility for pool1.x. I know it looks stupid, but could not find any smarter solution. Let me know if you have another suggestion, I'll look into it.